include/boost/geometry/algorithms/detail/buffer/buffered_piece_collection.hpp

```diff
@@ -191,8 +191,8 @@ struct buffered_piece_collection
 
         // 1: half, part of offsetted_rings
         segment_identifier first_seg_id;
-        int last_segment_index; // no segment-identifier - it is the same as first_seg_id
-        int offsetted_count; // part in robust_ring which is part of offsetted ring
+        signed_index_type last_segment_index; // no segment-identifier - it is the same as first_seg_id
+        signed_index_type offsetted_count; // part in robust_ring which is part of offsetted ring
```

Comment:
offsetted_count is a count but used in calculations with indexes. What do we normally do here?